Examples/ArcGISToolkitExamples/Misc/CalibrationView.swift

```diff
@@ -14,15 +14,21 @@
 
 import UIKit
 import ArcGIS
+import ArcGISToolkit
 
 /// A view displaying controls for adjusting a scene view's location, heading, and elevation. Used to calibrate an AR session.
 class CalibrationView: UIView {
-    
-    // The scene view displaying the scene.
-    private let sceneView: AGSSceneView
 
-    /// The camera controller used to adjust user interactions.
-    private let cameraController: AGSTransformationMatrixCameraController
+    /// Denotes whether to show the elevation control and label; defaults to `true`.
+    public var elevationControlVisibility: Bool = true {
+        didSet {
+            elevationSlider.isHidden = !elevationControlVisibility
+            elevationLabel.isHidden = !elevationControlVisibility
+        }
+    }
+    
+    /// The `ArcGISARView` containing the origin camera we will be updating.
+    private var arcgisARView: ArcGISARView!
```

Comment:
Correct me if I'm wrong, but this is only ever used to get at the origin camera, correct? If so, could you just initialize a calibration view with a camera instead? That way you don't need to introduce `ArcGISToolkit` as a dependency that is only used in passing.

Reply:
It also needs to set the `arcgisARView.originCamera`.